Dsl2 conversion (#82)
seqeralabs/nf-training-public: asciidocs/setup.adoc

image::gitpod.welcome.png[]

**The sidebar** allows you to customise your Gitpod environment and perform basic tasks (copy, paste, open files, search, git, etc.). Click the Explorer button to see which files are in this repository.

**The terminal** allows you to run all the programs in the repository, for example `nextflow` and `docker` are installed. 

> **The main window** allows you to view and edit files. Clicking on a file in the explorer will open it within the main window. We can also see the nf-training material in this window, but if you find the space too small you can open it separately in another browser window (https://training.seqera.io/). Finally, if you accidently close the Gitpod training window, you can reopen it by entering `gp preview https://training.seqera.io`.

Reviewer: ```suggestion
**The main window** allows you to view and edit files. Clicking on a file in the explorer will open it within the main window. We can also see the nf-training material in this window, but if you find the space too small you can open it separately in another browser window (https://training.seqera.io/). Finally, if you accidentally close the Gitpod training window, you can reopen it by entering `gp preview https://training.seqera.io`.
```

4. Saving your files in Gitpod to your local machine.

> To save your files, choose your file of interest, then either use the left/top hand side bar (File/Save As...) or use your keyboard shortcut to save as (On Mac: CMD, shift, s), then choose `Show local`. This will open up a explorer to choose where to save your file on your local machine.

Reviewer: ```suggestion
To save your files, choose your file of interest, then either use the left/top hand sidebar (File/Save As...) or use your keyboard shortcut to save as (On Mac: CMD, shift, s), then choose `Show local`. This will open up an explorer to choose where to save your file on your local machine.
```

This tutorial provides all the scripts, so don't worry if you have lost your environment. In the `nf-training` and `nf-training/scripts` directories, you can find the main scripts and individual snippets used in the tutorial.

> If you want to change git provider (between GitHub, GitLab and BitBucket), go to https://gitpod.io/integrations. You will need to login and deactive your current provider.

Reviewer: ```suggestion
If you want to change git provider (between GitHub, GitLab, and BitBucket), go to https://gitpod.io/integrations. You will need to log in and deactivate your current provider.
```
